pkg/profile/issuer/profile.go

```diff
@@ -125,6 +125,10 @@ func validateProfileRequest(pr *ProfileData) error {
 		return fmt.Errorf("supported vc contexts mandatory")
 	}
 
+	if pr.SupportsWACI && pr.IssuerID == "" {
+		return fmt.Errorf("issuer id mandatory for waci profiles")
```

Comment:
No, it is not mandatory, you can assign UUID if not set. Refer my comments from original PR.